src/main.rs

```diff
@@ -1,12 +1,12 @@
-#![feature(io, path, path_ext, std_misc)]
 extern crate glob;
-extern crate "rustc-serialize" as rustc_serialize;
+extern crate rustc_serialize;
 
 use std::collections::{HashMap, HashSet, hash_map};
 use std::io::prelude::*;
 use std::io::BufReader;
 use std::fs::File;
 use std::process::Command;
+use std::fs::metadata;
```

Comment:
Oh, could you instead import this via `use std::fs::{self, File};` two lines up?